src/login.rs

```diff
@@ -5,7 +5,10 @@ use std::convert::Infallible;
 use std::fmt::{Debug, Display};
 
 #[cfg(feature = "refreshing-token")]
-use {chrono::DateTime, chrono::Utc, std::time::Duration, thiserror::Error, tokio::sync::Mutex};
+use {chrono::DateTime, chrono::Utc, std::sync::Arc, std::time::Duration, thiserror::Error};
+
+#[cfg(feature = "refreshing-token")]
+use tokio::sync::Mutex;
```

Comment:
In this case, the line is too long to fit all the imports needed for `refreshing-token`. This would either have to be written as 
```rust
#[cfg(feature = "refreshing-token")]
use {chrono::DateTime, chrono::Utc, std::time::Duration, thiserror::Error, tokio::sync::Mutex};
#[cfg(feature = "refreshing-token")]
use std::sync::Arc;
```
or as 
```rust
#[cfg(feature = "refreshing-token")]
use {
    chrono::DateTime,
    chrono::Utc,
    thiserror::Error,
    tokio::sync::Mutex,
    {std::time::Duration, sync::Arc},
};
``` 
(which doesn't match the rest of the code style).

Reply:
I think the below style is perfectly fine, if `cargo fmt` formats it that way then it's acceptable.